invoice_creator.go

```diff
@@ -137,17 +137,19 @@ func (c *InvoiceCreator) Create(amtMSat int64, expiry time.Duration,
 
 	options = append(options, zpay32.Expiry(expiry))
 
+	if descHash != nil {
```

Comment:
Moving this up wasn't really necessary I think?

Reply:
No indeed, I changed my way of doing things along the way.

it's a vestige